java/src/jmri/jmrix/loconet/SlotManager.java

```diff
@@ -331,6 +333,23 @@ private void checkStaleSlots() {
         }
     }
 
+
+    /**
+     * Request slot data for 248 and 250
+     * Runs delayed
+     * <p>
+     * A call is trigger after the first slot response (PowerManager) received.
+     */
+    private void pollSpecialSlots() {
+        sendReadSlot(248);
+        try {Thread.sleep(10); }
+        catch (InterruptedException ex) {
+            // nothing can be done
+            return;
+        }
+        sendReadSlot(250);
+    }
```

Comment:
Perhaps better to make use of `jmri.util.runOnLayoutDelayed(@Nonnull ThreadAction ta, int delay)`, instead of `Timer.sleep`?

Reply:
May be. What happens in headless do we those runOnLayout / gui stuff available? I know we currently use a swing timer in headless mode, does it silently fail?